### PR TITLE
Added validation for only VSAN based storage policies and cleaned up output

### PR DESCRIPTION
**Pull request by thinkdavid**

### CODE REVIEW

File: Microsoft.AVS.Management/Microsoft.AVS.Management.psm1 [hunk 1]
```diff
@@ -193,7 +193,7 @@ function New-AvsLDAPIdentitySource {
 
 <#
     .Synopsis
-     Allow customers to add an LDAPS Secure external identity source (Active Directory over LDAP) for use with single sign on to vCenter. Prefaced by Connect-SsoAdminServer
+     Recommended: Allow customers to add an LDAPS Secure external identity source (Active Directory over LDAPS) for use with single sign on to vCenter.
```
Comment (et1975) [inline Microsoft.AVS.Management/Microsoft.AVS.Management.psm1:196]: This is confusing. Recommend ... Secure. But this one is not adding a secure/LDAPS source, is it?

Comment (thinkdavid) [reply Microsoft.AVS.Management/Microsoft.AVS.Management.psm1:196]: Yes, this was a mistake. Good catch

Comment (thinkdavid) [reply Microsoft.AVS.Management/Microsoft.AVS.Management.psm1:196]: Yeah, I'm not sure the best way to write this. But this is the one for adding a secure LDAPS source. I just remember feedback that they wanted to have it more explicitly stated that we really want customers to run this command and not the LDAP one

File: Microsoft.AVS.Management/Microsoft.AVS.Management.psm1 [hunk 2]
```diff
@@ -999,14 +999,15 @@ function Set-AvsDrsElevationRule {
...
-     Gets all the storage policies available to set on a VM 
+     Gets all the storage policies available to set on a VM.
 #>
 function Get-StoragePolicies {
     [AVSAttribute(3, UpdatesSDDC = $False)]
     
     $StoragePolicies
     try {
-        $StoragePolicies = Get-SpbmStoragePolicy -ErrorAction Stop
+        $JetstreamStoragePolicies = Get-SpbmStoragePolicy -ErrorAction Stop | Where-Object {$_.CommonRule -match '^jetdr.*'} | Select-Object Name
```
Comment (et1975) [inline Microsoft.AVS.Management/Microsoft.AVS.Management.psm1:1009]: Joe had a good idea for offering only policies in VSAN namespace, may let's go with that?

File: Microsoft.AVS.Management/Microsoft.AVS.Management.psm1 (hunk 1 above)
Comment (joesarabia) [reply Microsoft.AVS.Management/Microsoft.AVS.Management.psm1:196]: To align to other Microsoft docs/terminology & ldap cmdlet synopsis, I suggest: "Recommended: Allow customers to add a secure external identity source (Active Directory over LDAPS) for use with vCenter Single Sign-On."

File: Microsoft.AVS.Management/Microsoft.AVS.Management.psm1 [hunk 3]
```diff
@@ -586,10 +586,10 @@ function Add-GroupToCloudAdmins {
 
 <#
     .Synopsis
-     Remove a previously added external identity group from the cloud admin group
+     Remove a previously added group from an external identity from the vsphere CloudAdmins group
```
Comment (joesarabia) [inline Microsoft.AVS.Management/Microsoft.AVS.Management.psm1:589]: I suggest  just 'the CloudAdmins group' here for consistency with other descriptions.

File: Microsoft.AVS.Management/Microsoft.AVS.Management.psm1 (hunk 2 above)
Comment (joesarabia) [inline Microsoft.AVS.Management/Microsoft.AVS.Management.psm1:1009]: Suggest filtering by the vSAN namespace here via the Namespace parameter to exclude non-VSAN storage policies such as those that may be provided by 3rd parties.

File: Microsoft.AVS.Management/Microsoft.AVS.Management.psm1 (hunk 1 above)
Comment (et1975) [reply Microsoft.AVS.Management/Microsoft.AVS.Management.psm1:196]: Right, sorry, I was thinking I'm commenting on an unsecure version...

File: Microsoft.AVS.Management/Microsoft.AVS.Management.psm1 [hunk 4]
```diff
@@ -999,14 +999,15 @@ function Set-AvsDrsElevationRule {
...
+     Gets all the storage policies available to set on a VM.
 #>
 function Get-StoragePolicies {
     [AVSAttribute(3, UpdatesSDDC = $False)]
     
     $StoragePolicies
     try {
-        $StoragePolicies = Get-SpbmStoragePolicy -ErrorAction Stop
+        $JetstreamStoragePolicies = Get-SpbmStoragePolicy -ErrorAction Stop | Where-Object {$_.CommonRule -match '^jetdr.*'} | Select-Object Name
+        $StoragePolicies = Get-SpbmStoragePolicy -ErrorAction Stop | Select-Object Name, AnyOfRuleSets | Where-Object {$JetstreamStoragePolicies.Name -notcontains $_.Name}
```
Comment (joesarabia) [inline Microsoft.AVS.Management/Microsoft.AVS.Management.psm1:1010]: Suggest filtering by the vSAN namespace here via the Namespace parameter to exclude non-VSAN storage policies such as those that may be provided by 3rd parties.

File: Microsoft.AVS.Management/Microsoft.AVS.Management.psm1 [hunk 5]
```diff
@@ -999,14 +999,15 @@ function Set-AvsDrsElevationRule {
 
 <#
     .Synopsis
-     Gets all the storage policies available to set on a VM 
+     Gets all the storage policies available to set on a VM.
```
Comment (joesarabia) [inline Microsoft.AVS.Management/Microsoft.AVS.Management.psm1:1002]: If we do filter by Namespace as suggested below, we should say so here and in correspond Set, e.g. these cmdlets support modifying vSAN based storage policies on individual VMs, or similar.

**Review by et1975 — APPROVED** [review]

**Pull request merged**